src/mixins/with-resize-observer/with-resize-observer.js

```diff
@@ -32,9 +32,11 @@ export function withResizeObserver (ParentClass) {
         return;
       }
       const ro = new window.ResizeObserver(() => {
-        // NOTE: We could use entries[0].borderBoxSize.inlineSize but not supported in Chrome, Safari or polyfill
-        const { width } = this.getBoundingClientRect();
-        this._onResize({ width });
+        window.requestAnimationFrame(() => {
+          // NOTE: We could use entries[0].borderBoxSize.inlineSize but not supported in Chrome, Safari or polyfill
+          const { width } = this.getBoundingClientRect();
+          this._onResize({ width });
+        });
```

Comment:
thanks @pdesoyres-cc apparently it does not break anything and it does fix a lot of `resize observer loop limit exceeded issue` occurrences so why not ?

Anyway, this commit might be dropped if we implement the lit resize observer before the merge.